From 8bfb3f0123ef6519d70baaf9c8ac2ef3cf1b86ff Mon Sep 17 00:00:00 2001
From: Alex Gonzalez <alex.gonzalez@digi.com>
Date: Mon, 21 Jan 2013 17:07:18 +0100
Subject: [PATCH] redpine: allow to build with gcc-4.7

Otherwise it fails with:

error: inlining failed in call to always_inline 'ganges_bufferfull_acquire_lock': function body not available
error: inlining failed in call to always_inline 'ganges_bufferfull_release_lock': function body not available

Signed-off-by: Javier Viguera <javier.viguera@digi.com>
Signed-off-by: Alex Gonzalez <alex.gonzalez@digi.com>
---
 RS.GENR.LNX.SD_NON_GPL/include/Linux/ganges_mgmt.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/RS.GENR.LNX.SD_NON_GPL/include/Linux/ganges_mgmt.h b/RS.GENR.LNX.SD_NON_GPL/include/Linux/ganges_mgmt.h
index 2eb31f5..aa789a7 100644
--- a/RS.GENR.LNX.SD_NON_GPL/include/Linux/ganges_mgmt.h
+++ b/RS.GENR.LNX.SD_NON_GPL/include/Linux/ganges_mgmt.h
@@ -1093,10 +1093,10 @@ INT32 ganges_start_per_tx(PGANGES_CLIENT Adapter,UINT8 *ptr);
 GANGES_STATUS ganges_do_continuous_send(PGANGES_CLIENT Adapter, UINT16  mode);
 
 VOID ganges_fsm_compute_snr(PGANGES_CLIENT Adapter,PScanConfirm scanConfirm,UINT8 mode);
-inline VOID ganges_bufferfull_acquire_lock(
+VOID ganges_bufferfull_acquire_lock(
          DWORD (*fptr)(PVOID sem,INT delay_msec),
          PGANGES_CLIENT Adapter);
-inline BOOLEAN
+BOOLEAN
 ganges_bufferfull_release_lock
   (
     BOOLEAN (*fptr)(PVOID sem),
